maps_red/ticketing-bundle/0.6/src/Repository/TicketHistoryRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\TicketHistory;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+use Maps_red\TicketingBundle\Repository\TicketHistoryRepository as BaseTicketHistoryRepository;
+
+/**
+ * @method TicketHistory|null find($id, $lockMode = null, $lockVersion = null)
+ * @method TicketHistory|null findOneBy(array $criteria, array $orderBy = null)
+ * @method TicketHistory[]    findAll()
+ * @method TicketHistory[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class TicketHistoryRepository extends BaseTicketHistoryRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, TicketHistory::class);
+    }
+
+//    /**
+//     * @return TicketHistory[] Returns an array of TicketHistory objects
+//     */
+    /*
+    public function findByExampleField($value)
```

Comment:
Do we really need all this comments with example code?

It is just a simple query builder. I suggest removing it.